File: python/httpcall.py
import uvicorn
import aiohttp
from fastapi import FastAPI

app = FastAPI()

@app.get("/")
async def helloworld():
    async with aiohttp.ClientSession() as sess:
        async with sess.get("https://postman-echo.com/get?foo1=bar1&foo2=bar2") as resp:
            return await resp.text()

if __name__ == "__main__":
    uvicorn.run(app, host="0.0.0.0", port=8000)

File: nodejs/httpcall.js
const fastify = require('fastify')({ logger: true })
const https = require("https")

fastify.get('/', (request, reply) => {
    let data;
    https.get("https://postman-echo.com/get?foo1=bar1&foo2=bar2", resp => {
        resp.on("data", chunk => data += chunk);
        resp.on("end", () => reply.code(200).send(data));
        resp.on("error", (err) => console.log(err));
    });
})

const start = async () => {
    try {
        await fastify.listen(3000)
        fastify.log.info(`server listening on ${fastify.server.address().port}`)
    } catch (err) {
        fastify.log.error(err)
        process.exit(1)
    }
}
start()